packages/pangea-node-sdk/CHANGELOG.md

```diff
@@ -7,6 +7,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+### Added
+
+- Vault v2 APIs
```

Comment:
nitpick: this might be better as a "Changed" item so that existing Vault users know that their current usage will need to change to work with Vault v2.

Reply:
Fixed!